tests/acceptance/16_cf-serverd/serial/copy_missing_ok.cf.sub

```diff
@@ -0,0 +1,55 @@
+#######################################################
+#
+# Tries to copy using TLS (which is default now), from two servers: one
+# with the default TLS ciphers list and another with a non-default very
+# restricted one.
+#
+# It should fail since we are setting "tls_ciphers" to a cipher missing
+# in both servers.
+#
+#######################################################
+
+body common control
+{
+    inputs => { "../../default.cf.sub" };
+    bundlesequence  => { default("$(this.promise_filename)") };
+}
+
+bundle agent init
+{
+}
+
+bundle agent test
+{
+    files:
+        "$(G.testdir)/missing_ok"
+            classes => classes_generic("test"),
+            copy_from       => dcs_remote_cp_X("does_not_exists", "localhost", "true");
```

Comment:
By using "localhost" here, the agent uses a shortcut and copies from the local directory without ever hitting the test server. I tested it with:
```
./testall --no-clean --extraclasses=debug_mode 16_cf-serverd/serial/copy_missing_ok.cf
```

which runs the server in debug mode and then I examined the logs; no incoming connections.